tasks/tls_signed.yml

```diff
@@ -1,32 +1,57 @@
 ---
-- name: "[SIGNED TLS] - Define signed certificate path"
+- name: "[SIGNED TLS] - Define certificate-file locations as abolute path"
   ansible.builtin.set_fact:
-    nextcloud_tls_cert_file: "{{ nextcloud_tls_cert | default(\"/etc/ssl/\" + nextcloud_instance_name + \".crt\") }}"
+    cert_path: "/etc/ssl/"
 
-- name: "[SIGNED TLS] - Define signed certificate's key path"
+- name: "[SIGNED TLS] - Define signed certificate-file with abolute path"
   ansible.builtin.set_fact:
-    nextcloud_tls_cert_key_file: "{{ nextcloud_tls_cert_key | default(\"/etc/ssl/\" + nextcloud_instance_name + \".key\") }}"
+    nextcloud_tls_cert_file: "{{ nextcloud_tls_cert | default(cert_path + nextcloud_instance_name + \".crt\") }}"
 
-- name: "[SIGNED TLS] - Certificate is on the host"
+- name: "[SIGNED TLS] - Define signed certificate's key-file with absolute path"
+  ansible.builtin.set_fact:
+    nextcloud_tls_cert_key_file: "{{ nextcloud_tls_cert_key | default(cert_path + nextcloud_instance_name + \".key\") }}"
+
+- name: "[SIGNED TLS] - Define certificate chain-file with absolute path"
+  ansible.builtin.set_fact:
+    nextcloud_tls_chain_file: "{{ nextcloud_tls_chain_file | default(cert_path + nextcloud_instance_name + \".pem\") }}"
+  when: nextcloud_tls_src_chain is defined
```

Comment:
Do you want to put `cert_path` in defaults ? looks very much like a default value :yum: 
Also I found out that set_facts module can set multiple variables in one task.
Could you kindly process that ? this will greatly simplify the role
Thank you.

Reply:
That would make sense, yes. Where should I place it, please advice.

Reply:
for  `cert_path` in defaults/main.yml , but maybe you 'll need to rename it to make it consistent with the other cert related variables

Reply:
Changed and added to README.